### ISSUE TITLE
It removes comments and licences

### ISSUE THREAD

**Issue by alvarotrigo** [caiguanhao/js-obfuscator#14]
Right not it seems to remove all comments with their licenses.
It would be nice to have something like `gulp-uglify` does. ` preserveComments: 'license'`

**Comment by caiguanhao** [comment]
No. This plugin uses the free version of Javascript Obfuscator. It doesn't have the feature you mentioned. However, it has a feature called "[Keep Header Comment](http://javascriptobfuscator.com/Javascript-Obfuscator.aspx)" in the advanced version.
